src/TreeMenu/__tests__/TreeMenu.test.tsx

```diff
@@ -72,6 +92,19 @@ describe('TreeMenu', () => {
     expect(wrapper).toMatchSnapshot();
     expect(highlightedElement.props.className).toContain('rstm-tree-item--active');
   });
+  it('should work with an alternative key separator', () => {
+    const wrapper = mount(
+      <TreeMenu
+        data={mockDataWithSlashes}
+        openNodes={['item2/a']}
+        keySeparator=";"
```

Comment:
hi @PRGfx , thanks for the PR! It's indeed a useful prop!
Is it possible to make this test case more specific by setting `openNodes` to `item2/a;b` (and `b` needs `nodes` as well)? So that it can fully demonstrate the use case of this new prop 🙂